stackdriver-debugger/src/com/google/cloud/tools/intellij/stackdriver/debugger/CloudDebugRunConfiguration.java

```diff
@@ -148,7 +159,7 @@ public void setProcessState(@Nullable CloudDebugProcessState processState) {
   public RunProfileState getState(
       @NotNull Executor executor, @NotNull ExecutionEnvironment environment) {
     if (processState == null) {
-      return new CloudDebugProcessState(null, null, cloudProjectName, null, getProject());
+      return new CloudDebugProcessState(googleUsername, null, cloudProjectId, null, getProject());
```

Comment:
same comment here for the nulls (named param comments so its easier to read quickly).